docker-compose.yml

```diff
@@ -0,0 +1,20 @@
+mysql:
+    extends:
+      file: common-compose.yml
+      service: mysql
+    links:
+      - consul:consul
+
+mysql_replica:
```

Comment:
Rather than doing leader "election", we can rely on the fact that `docker-compose` starts one node first. The decision tree looks like this:

- during `onStart` the nodes will ask Consul if there is a primary
  - no? the node will write a key to Consul marking itself as primary. was DB initialized?
    - no? init the DB and start mysqld
    - yes? stop acting as a replica and start mysqld
  - yes? ok, is the primary healthy?
    - no? halt and catch fire
    - yes? set that primary as its source and start mysqld

If we need to promote a replica, we can clear the key in Consul and just restart a replica, then `docker exec` a command to the replicas to `CHANGE MASTER` to the new primary.